### DISCUSSION TITLE
Events only pull from one Calendar?

### DISCUSSION THREAD

**Discussion by camrun91** [ApollosProject/apollos-templates#438] [Q&A] [answered]
I am looking at adding Events to our search. However, the Apollos Api for Rock only pulls from one calendar. We use a few different calendars currently. I want to eventually move to a single calendar but if we could add support for multiple calendars that would be good.

**Comment by redreceipt** [comment, marked as answer]
We'd welcome support for multiple calendars

**Reply by camrun91** [reply]
OK I have this working roughly in our app. I can look at submitting a PR.
